src/common/KernelBase.hpp

```diff
@@ -412,6 +422,15 @@ class KernelBase
      getCout() << "\n KernelBase: Unimplemented Kokkos variant id = " << vid << std::endl;
   }
 #endif
+#if defined(RAJA_ENABLE_SYCL)
+  virtual void runSyclVariant(VariantID vid, size_t tune_idx)
+  {
+     getCout() << "\n KernelBase: Unimplemented Sycl variant id = " << vid << std::endl;
+  }
+  static sycl::queue* qu;
+  static camp::resources::Resource sycl_res;
```

Comment:
Why is the sycl resource handled differently than the cuda and hip resources?

Reply:
We should probably always grab the queue from the resource when we need it, then we can't accidentally use two different queues.

Reply:
Or always use the getSyclResource function instead of storing it here.